src/app/app.component.ts

```diff
@@ -86,6 +86,7 @@ export class AppComponent {
 
 					// Step number must be incremental after the sorting
 					this.testSuites.forEach((suite) => {
+						suite.testCases.sort((a, b) => a.start < b.start ? -1 : a.start < b.start ? 1 : 0);
```

Comment:
Are you sure about this line of code?

Reply:
Yes, this line is the fixing for the issue #22. It's required to sort the steps in the same order as executed.

Reply:
The reason is clear, but the code is correct or it should be:
suite.testCases.sort((a, b) => a.start < b.start ? -1 : a.start > b.start ? 1 : 0);
I mean change the second **a.start < b.start** by **a.start > b.start** ???

Reply:
You're right. I've just uploaded the fixing.